src/v/kafka/requests/fetch_request.cc

```diff
@@ -118,26 +118,38 @@ void fetch_request::decode(request_context& ctx) {
     }
 }
 
+std::ostream&
+operator<<(std::ostream& o, const fetch_request::forgotten_topic& t) {
+    return ss::fmt_print(o, "{{topic {} partitions {}}}", t.name, t.partitions);
```

Comment:
use the `fmt::format_to` instead of `fmt_print`

Reply:
you mean `fmt::print` ?

Reply:
yeah i thinkso print_to or whatever is called. look inside fmt_print